model/message.go

```diff
@@ -16,11 +16,13 @@ package model
 
 // Message constants
 const (
-	CmdShell = "shell"
+	TypeShell = "shell"
 )
 
 // Message represents a message between the device and the backend
 type Message struct {
-	Cmd  string `json:"cmd" msgpack:"cmd"`
-	Data []byte `json:"data" msgpack:"data"`
+	Type      string `json:"type" msgpack:"type"`
+	SessionID string `json:"session_id" msgpack:"session_id"`
+	Status    int    `json:"status" msgpack:"status"`
```

Comment:
when I was testing with gui I got terrible errors when "status" field was used from the js, so [I changed to: status_code](https://github.com/mendersoftware/mender-shell/pull/12/commits/3a456c0ed233a8055a288326f80baa3d73bcc3d9#diff-bc40df7974ec36bb1274390437a11285e121646b180b6bbc0acdd1b24e1bfb75R50) if you have no problems with "status" when marshalling in js from the ui I can change back to "status".

Reply:
it works just fine...